content/en/blog/_posts/2019-07-09-some-apis-are-being-deprecated.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: blog
+title: 'APIs Deprecating Soon: Here\'s What You Need to Know'
```

Comment:
Might need to be `”APIs deprecating Soon: Here’s What You Need To Know”`

Reply:
nit, these were deprecated ~years ago... they're being removed in 1.16

Reply:
Changed accordingly.